inadarei/justgo-microservice
# Simplifying for dep

## Motivation
Start using $GOPATH in a way that go dep wants us and stop fighting it. Simplifies everything else.

## Steps to Test:
1. Check a fresh copy out
1. make sure you clean-out all existing justgo containers and justgo images
1. run `make`
1. run `make ps` and `make logs`. Make sure server comes up just fine
1. edit server/server.go, e.g. message on line 24, to test hot-reloading
1. Make sure server hot reloads properly
